Homework_1.py

```diff
@@ -34,10 +34,17 @@
     else:
         odd_list.append(value) # if remainder of the division by 2 is not 0, then the number is odd and add it in odd_list
 
-even_avg = sum (even_list)/len (even_list) # sun of values in the even_list devided by the number of values in the even_list
-odd_avg = sum (odd_list)/len (odd_list) # sun of values in the odd_list devided by the number of values in the odd_list
-
-# 4. print both average result in console
-print ('The average result for even list is ', '%.3f' % even_avg) # print the average value with 3 numbers after comma
-print ('The average result for odd list is ', '%.3f' % odd_avg) # print the average value with 3 numbers after comma
+try: # exception of devision by zero
+    # even_list=[] # check empty list
+    even_avg = sum (even_list)/len (even_list) # sun of values in the even_list devided by the number of values in the even_list
+    print('The average result for even list is ', '%.3f' % even_avg)  # print the average value with 3 numbers after comma
+except:
```

Comment:
I recommend to handle particular exceptions separately. So, in this example you are expecting ZeroDivisionError, let's handle this particular exception:)